docs/development/getting_started.rst

```diff
@@ -0,0 +1,186 @@
+.. _development_getting_started:
+
+===============
+Getting started
+===============
+
+The project is developed in Python using the `Django framework`_. There are 3
+sections below, focussing on developers, running the project using Docker and
+hints for running the project in production.
+
+.. _Django framework: https://www.djangoproject.com/
+
+Installation
+============
+
+Prerequisites
+-------------
+
+You need the following libraries and/or programs:
+
+* `Python`_ 3.7 or above
+* Python `Virtualenv`_ and `Pip`_
+* `PostgreSQL`_ 10 or above, with the `PostGIS-extension`_
+* `Node.js`_
+* `npm`_
+
+.. _Python: https://www.python.org/
+.. _Virtualenv: https://virtualenv.pypa.io/en/stable/
+.. _Pip: https://packaging.python.org/tutorials/installing-packages/#ensure-pip-setuptools-and-wheel-are-up-to-date
+.. _PostgreSQL: https://www.postgresql.org
+.. _PostGIS-extension: https://postgis.net/
+.. _Node.js: http://nodejs.org/
+.. _npm: https://www.npmjs.com/
+
+
+Getting started
+---------------
+
+Developers can follow the following steps to set up the project on their local
+development machine.
+
+1. Navigate to the location where you want to place your project.
+
+2. Get the code:
+
+   .. code-block:: bash
+
+       $ git clone git@github.com:open-zaak/open-zaak.git
+       $ cd open-zaak
+
+3. Install all required libraries:
+
+   .. code-block:: bash
+
+       $ virtualenv env
+       $ source env/bin/activate
+       $ pip install -r requirements/dev.txt
+
+4. Install the front-end CLI tool `gulp`_ if you've never installed them
+   before and install the frontend libraries:
+
+   .. code-block:: bash
+
+       $ npm install -g gulp
+       $ npm install
+       $ gulp sass
+
+5. Activate your virtual environment and create the statics and database:
+
+   .. code-block:: bash
+
+       $ source env/bin/activate
+       $ python src/manage.py collectstatic --link
+       $ python src/manage.py migrate
+
+6. Create a superuser to access the management interface:
+
+   .. code-block:: bash
+
+       $ python src/manage.py createsuperuser
+
+7. You can now run your installation and point your browser to the address
+   given by this command:
+
+   .. code-block:: bash
+
+       $ python src/manage.py runserver
+
+8. Create a .env file with database settings. See ``.env-example`` for an
```

Comment:
The `.env` should happen before runserver/migrate if you're going to use it, but you don't _need_ it for dev.